.github/CONTRIBUTING.rst

```diff
@@ -214,7 +214,7 @@ We use `sphinx`_ to generate static HTML docs. To build them, first make sure yo
 
 .. code-block:: bash
 
-   $ pip install -r docs/requirements-docs.txt
+   $ pip install .[docs]
```

Comment:
just a thought: we already tell to install all dev dependencies which installs doc dependencies, so there's actually no need of this step